src/state/reducer.ts

```diff
@@ -4,13 +4,20 @@ import storage from 'redux-persist/lib/storage';
 
 import alertReducer from './alert/reducer';
 import authReducer from './auth/reducer';
+import restockReducer from './restock/restockReducer';
 
 const persistConfig = {
   key: 'root',
   storage: storage,
   whitelist: ['auth', 'alert'],
```

Comment:
Forgot to whitelist

Reply:
I don't want to whitelist this. There's no need to persist this data.